.gitignore

```diff
@@ -86,6 +86,7 @@ celerybeat-schedule
 # virtualenv
 .venv/
 venv/
+venv_mtgpl/
```

Comment:
Jeśli potrzebujesz zrobić taki wpis, to masz źle ustawiony virtualenv. Sugeruję postawić virtualenvwrapper i tworzyć środowisko z jego poziomu przez `mkvirtualenv` - tworzone środowisko jest poza repozytorium i nie trzeba robić tego typu wpisów. Potem tylko `workon [środowisko]` i jedziesz :smiley: